### PR TITLE
InventoryEmeraldCountFeature: Fix categories showing up even though there is not enough emeralds

### PR DESCRIPTION
**Pull request by kristofbolyai**

### CODE REVIEW

File: common/src/main/java/com/wynntils/features/user/inventory/InventoryEmeraldCountFeature.java [hunk 1]
```diff
@@ -123,7 +123,7 @@ private void renderTexturedCount(PoseStack poseStack, int emeralds) {
 
         for (int i = emeraldAmounts.length - 1; i >= 0; i--) {
             String emeraldAmount = emeraldAmounts[i];
```
Comment (magicus) [inline common/src/main/java/com/wynntils/features/user/inventory/InventoryEmeraldCountFeature.java]: Why is this a string and not an integer?

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/features/user/inventory/InventoryEmeraldCountFeature.java]: It is formatted in decimal and stuff

Comment (magicus) [reply common/src/main/java/com/wynntils/features/user/inventory/InventoryEmeraldCountFeature.java]: So is the check for "0" always valid then? Maybe it is a french zero? ;-)

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/features/user/inventory/InventoryEmeraldCountFeature.java]: It should be, yes.

Comment (magicus) [reply common/src/main/java/com/wynntils/features/user/inventory/InventoryEmeraldCountFeature.java]: I think what I am saying is that we should not have two different ways to represent "no value".

**Review by magicus — APPROVED** [review]
Thanks!

**Pull request merged**